.github/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,105 @@
+# Salesforce Open Source Community Code of Conduct
+
+## About the Code of Conduct
+
+Equality is a core value at Salesforce. We believe a diverse and inclusive
+community fosters innovation and creativity, and are committed to building a
+culture where everyone feels included.
+
+Salesforce open-source projects are committed to providing a friendly, safe, and
+welcoming environment for all, regardless of gender identity and expression,
+sexual orientation, disability, physical appearance, body size, ethnicity, nationality,
+race, age, religion, level of experience, education, socioeconomic status, or
```

Comment:
```suggestion
sexual orientation, disability, physical appearance, body size, ethnicity, nationality, 
race, age, religion, level of experience, education, socioeconomic status, or 
```

👁️‍🗨️ note: No blocker but IIRC the template repo trails some lines with spaces. We might match this exact to make future `diff` easier, but of course we can follow up if that's needed!

https://github.com/salesforce/oss-template/blob/main/CODE_OF_CONDUCT.md?plain=1#L11-L12

Reply:
Thanks! I grabbed these files from another template, so that's probably where the whitespace came from.